docs/promscale_alerting.md

```diff
@@ -0,0 +1,226 @@
+# Promscale Alerting
+
+This file contains alerting rules which are symptom based. We recommend monitoring your Promscale deployments
+and include the following rules as part of your alerting environment.
+
+## Rules
+
+Copy the below in a `promscale_alerts.yaml` file.
+
+```yaml
+groups:
+- name: Promscale
+  rules:
+  - alert: PromscaleDown
+    expr: absent(up{job=~".*promscale.*"})
+    for: 2m
+    labels:
+      severity: critical
+      component: connector
+    annotations:
+      summary: Promscale is down
+      description: No Promscale instance was found.
+# Ingest
```

Comment:
```suggestion
- name: promscale-ingest
```

Instead of using comments in yaml code (which are lost in prometheus), we can put those alerts in separate groups.

Added bonus of doing this would be parallelization of alert expression execution in prometheus (groups are run in parallel, the content of a group is run sequentially).